pkgs/development/python-modules/js2py/default.nix

```
@@ -1,20 +1,18 @@
 { lib
-, fetchFromGitHub
+, fetchPypi
 , buildPythonPackage
 , tzlocal
 , six
 , pyjsparser
 }:
 
 buildPythonPackage rec {
-  pname = "js2py";
-  version = "0.71";
+  pname = "Js2Py";
```

`pname` should be normalized as it was.

```suggestion
  pname = "js2py";
```

---

Done.